packages/circuit-ui/package.json

```diff
@@ -95,8 +95,8 @@
     "@emotion/styled": "10.x",
     "@emotion/stylis": "0.8.x",
     "@sumup/collector": "1.x",
-    "@sumup/design-tokens": ">=1.0.0 <3.0.0",
-    "@sumup/icons": "1.x",
+    "@sumup/design-tokens": ">=3.0.0",
+    "@sumup/icons": "1.7.1",
```

Comment:
Not sure if these are what we want in peer dependencies

Reply:
I think it's fine. In the future, I wouldn't treat a minor peer dependency bump as a breaking change.